backends/events/sdl/sdl-events.h

```diff
@@ -59,7 +59,9 @@ class SdlEventSource : public Common::EventSource {
 
 	struct KbdMouse {
 		int16 x, y, x_vel, y_vel, x_max, y_max, x_down_count, y_down_count;
+		static const int16 multiplier = 16;
```

Comment:
What I had in mind was actually using an enum:
```
enum {
	kKbdMouseMultiplier = 16
};
```

But a define or a const int is fine as well I think.